Beaver Notes 3.9.0

Daniele-rolli opened the pull request:
## Pull Request Type
- [x] **Feature/Component** (New feature or component)
- [x] **Bug Fix** (Fixes an issue or bug)
- [ ] **Minor Change** (Documentation, README updates, or small UI tweaks)

## Checklist (For Features/Bug Fixes)

- [x] Does this PR address the **intended feature/bug** without any unintended changes?
- [x] Have all **new components/features** been tested for functionality?
- [x] Have all **bugs been fixed** and verified?
- [x] Is the **UI/UX** consistent across different screen/window sizes?
- [x] Have **translations** been updated for all new user-facing components?
- [x] Have you confirmed that the **same work has been applied / will be applied to the mobile app** (if applicable)?
- [x] Is this PR ready for release, with appropriate versioning?

## Description of Changes
### **🎉 New**
- Accent Colors #276 
### **🐞 Fixes**
- UI size #274 
- Paste #275 
- PDF export
- Windows Portable Path #273 
- Sharing dialog (🐧,🪟)
- Improved Commands menu
### **⭐️ Improvements**
- Better editor search
- Better file handling

*edit: specified editor search*

bigshans commented:
1. The exported PDF includes the cursor.
![图片](https://github.com/user-attachments/assets/678d305f-7a36-4cbf-b20d-ce6175336bcb)
2. Pasting is abnormal. First, pasting text will be duplicated. Second, the next pasted content will directly replace the previous content.
![Peek 2025-02-10 21-06](https://github.com/user-attachments/assets/d592bfe9-02ae-42d2-a3eb-d1a995d2437d)

bigshans commented:
It is fixed but the formatting of the HTML paste is lost.

Daniele-rolli commented:
@bigshans Finally figured it out! Basically, it now detects both Markdown and HTML and handles them accordingly, while plain text is handled by Tiptap. Feel free to try it out and let me know what you think!

bigshans commented:
I updated my solution. If the clipboard stores the HTML data, Tiptap can handle it. We only deal with the plain text.

bigshans commented:
You can check it out and tell me what you think too.

Daniele-rolli commented:
@bigshans Pasting works great! I've added translations for the callout and fixed some bugs and minor issues. Let me know what you think. I believe we should be ready for the release.

bigshans commented:
I updated the Chinese translation. I think we are ready for it.
